### PR TITLE
tests-invoke: Stringify returncode

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

**Comment by marusak** [comment]
https://209.132.184.41:8493/logs/pull-13433-20200122-141414-fe0cc760-cockpit-project-cockpit--ubuntu-1804/log example failure

File: tests-invoke [hunk 1]
```diff
@@ -268,7 +268,7 @@ class PullTask(object):
                 else:
                     time.sleep(60)
             ret = p.returncode
-            sys.stderr.write("Test run finished, return code: {0}\n", ret)
+            sys.stderr.write("Test run finished, return code: {0}\n", str(ret))
```
Comment (martinpitt) [inline tests-invoke]: No, it needs to call `.format(ret)`, or use `'code: %s' % ret`. The stringification is totally not the problem.

Comment (Gundersanne) [reply tests-invoke]: sorry my brain is fried :|

File: tests-invoke [hunk 2]
```diff
@@ -268,7 +268,7 @@ class PullTask(object):
                 else:
                     time.sleep(60)
             ret = p.returncode
-            sys.stderr.write("Test run finished, return code: {0}\n", ret)
+            sys.stderr.write("Test run finished, return code: {0}\n".format(ret))
```
Comment (martinpitt) [inline tests-invoke]:
```
❱❱❱ python3 -c 'import sys; sys.stderr.write("Test run finished, return code: {0}\n".format(1))'
Test run finished, return code: 1
```

**Pull request merged**